abbyy_to_epub3/__init__.py

```diff
@@ -1,4 +1,4 @@
 
 __title__ = 'abbyy_to_epub3'
-__version__ = '1.6'
+__version__ = '1.61'
```

Comment:
QUESTION: Did you want to use semantic versioning?

Reply:
resolved